src/app/modules/main/chat_section/module.nim

```diff
@@ -772,7 +772,11 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
 
   let contactDetails = self.controller.getContactDetails(message.`from`)
   let renderedMessageText = self.controller.getRenderedText(message.parsedText)
-  let plainText = singletonInstance.utils.plainText(renderedMessageText)
+  
+  var plainText = singletonInstance.utils.plainText(renderedMessageText)
+  if message.contentType.ContentType == ContentType.Image: 
+    plainText = "Image received"
```

Comment:
This is untranslatable; should happen in QML

Reply:
Good point, but I saw other updated text in the Nim code (which can be wrong)... @jrainville how do you usually handle such situations?

Reply:
I'm afraid we don't currently have any solution at all for this; let's find something :)

Reply:
solved 😄